### ISSUE TITLE
Android: Duplicate class org.walleth.khex.HexFunKt found in modules KHex-0.6.jar (com.github.komputing:KHex:0.6) and khex-0.6.jar (com.github.walleth:khex:0.6)

### ISSUE THREAD

**Issue by iwashi1t** [uport-project/react-native-uport-signer#21]
Thank you for creating this wonderful library.

I installed & linked `react-native-uport-signer: 1.3.3` and built a debug, the following error occurred.

```
> Task :app:checkDebugDuplicateClasses FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:checkDebugDuplicateClasses'.
> 1 exception was raised by workers:
  java.lang.RuntimeException: Duplicate class org.walleth.khex.HexFunKt found in modules KHex-0.6.jar (com.github.komputing:KHex:0.6) and khex-0.6.jar (com.github.walleth:khex:0.6)

  Go to the documentation to learn how to <a href="d.android.com/r/tools/classpath-sync-errors">Fix dependency resolution errors</a>.
```

I checked the gradle dependencies and it seems that both `react-native-uport-signer` and `uport-android-signer` have `kotlin-common:signer-common`, so when I removed the former, the build passed.

```
dependencies {
    implementation 'com.facebook.react:react-native:+'

    api "com.github.uport-project:uport-android-signer:0.3.1"
    api "com.github.uport-project.kotlin-common:signer-common:0.1.1"  // removed this line
}
```

Is the above line necessary?

Thank you.

**Comment by mirceanis** [comment]
Thanks for reporting this.

We have stumbled upon similar errors in the past but it would seem that not all instances of it have been put out. Without knowing your particular configuration, it's hard to tell if removing that line will fix it for everyone or cause compilation issues.

The root cause is a naming conflict that appears deeper down in the dependency tree on Android (KHex).
It will be fixed on its own when all the dependencies in the tree that rely on it update to the new coordinates.

Until then, there are workarounds.

The quickest is to add the following to your `android/app/build.gradle` file:
```
configurations.all {
    exclude group: "com.github.walleth"
}
```

Please try this workaround and let us know.
Is it possible to share a project that has is manifesting this failure?

**Comment by mirceanis** [comment]
this should be fixed in v1.3.4
